internal/handlers/jsonb1/where.go

```diff
@@ -266,6 +266,8 @@ func wherePair(key string, value any, p *pg.Placeholder) (sql string, args []any
 	default:
 		// {field: value}
 		switch value.(type) {
+		case int32, int64:
+			sql = fmt.Sprintf("jsonb_typeof(_jsonb->%[1]s) = 'object' AND _jsonb->%[1]s @> to_jsonb('{\"$l\":\"%v\"}'::jsonb) OR _jsonb->%[1]s @>", p.Next(), value)
```

Comment:
I think that should be two different `case`s at the very least.

And `'{\"$l\":\"%v\"}'` is definitly not secure. We use `scalar` function below, so I think it could work there too

Reply:
@AlekSi Why it is not secure? the value passed to the Sprintf() is checked to be a number.